recipes/expo-turtle-build.md

```diff
@@ -0,0 +1,102 @@
+# (Expo) Build using Turtle CLI
+
+## Description
+
+Publish an app to Expo's servers and build an iOS App Store .ipa and Android .aab files from your Expo project using [Turtle CLI](https://docs.expo.dev/classic/turtle-cli/).
```

Comment:
NIT: `.ipa` and .aab highlight?

Reply:
Added code format in: https://github.com/bitrise-io/workflow-recipes/pull/1/commits/aa8469ff2ff570219625c45ff41c93d815e1544b